### DISCUSSION TITLE
Is there a way to set a custom name or label for layouts ?

### DISCUSSION THREAD

**Discussion by lyndhurst** [qtile/qtile#4684] [Q&A] [answered]
Hi,

I think I looked all over the doc for a `label` or `name` kind of parameter to add to `Layout` objects, and did not find it, but every time I fall on the `config file > variables > groups` page, the following quote leaves me wondering:

>layout:
>    The name of default layout for this group (e.g. "max"). This is the name specified for a particular layout in config.py or if not defined it defaults in general to the class name in all lower case.

When I read this part: "This is the name specified for a particular layout in `config.py`" I can't help but feel like I am missing something. It is not a critical feature that I need, it could be useful for the `CurrentLayout` widget maybe, but I am curious about it.

Thanks for letting me know if I am just "over-interpreting" the documentation.

**Comment by elParaguayo** [comment, marked as answer]
Just add `name="my_layout_name"` to the layout's config.

**Reply by lyndhurst** [reply]
This is something I could have actually tried without asking for help. Thank you and sorry about that.

**Reply by elParaguayo** [reply]
No problem at all. It's not widely documented and I don't recall someone asking before.
